stable/anchore-engine/values.yaml

```diff
@@ -308,6 +308,10 @@ anchoreGlobal:
       failureThreshold: 3
       successThreshold: 1
 
+  # using the preupgrade hook will create a service account that will use
```

Comment:
I would add a note here that this is useful for deployments using `helm upgrade --wait` or ArgoCD. Also mention that it create the role/rolebinding as well as the SA, maybe even mention the permissions granted to that SA as well.

Reply:
+1 on adding the note about `helm upgrade --wait`

Reply:
added